experimental/telemetry/tracker.go

```diff
@@ -93,6 +93,6 @@ func (t *tracker) TrackEvent(event string, properties map[string]interface{}) {
 }
 
 func (t *tracker) TrackUserEvent(event, userID string, properties map[string]interface{}) {
-	properties["UserID"] = userID
+	properties["UserActualID"] = userID
```

Comment:
NPS uses `user_actual_id'. Should that also be used here? https://github.com/mattermost/mattermost-plugin-nps/blob/75b671a998901e00253fe04b840047eccb74e3a4/server/segment.go#L61

Reply:
RudderStack ends up transforming `UserActualID` into `user_actual_id` so in effect, `UserActualID` == `user_actual_id`

Reply:
Does it make sense to use `user_actual_id` here for clarity?